irods/test/user_group_test.py

```diff
@@ -604,6 +605,12 @@ def test_set_and_query_user_quota(self):
             if bob:
                 ses.users.remove(bob.name)
 
+     def test_we_cannot_overwrite_irodsA_without_overwrite_option_true__issue_635()
+         self.assertTrue(os.path.isfile(os.path.expanduser('~/.irods/.irodsA')))
+         with self.assertRaises(client_init.irodsA_already_exists):
+             client_init.write_native_credentials_to_secrets_file('somevalue',overwrite=False)
+         with self.assertRaises(client_init.irodsA_already_exists):
+             client_init.write_pam_credentials_to_secrets_file('somevalue',overwrite=False)
```

Comment:
Codacy says this indentation is wrong... looks like 5 spaces were used on L608 instead of 4, but the rest is fine relative to that.

Reply:
also no self parameter!